### ISSUE TITLE
[Feature request] Option to show all tokens without tapping

### ISSUE THREAD

**Issue by error401de** [helloworld1/FreeOTPPlus#68]
It would be awesome to get an option to show all tokens, right now I need to tap it as it's hidden by default.
Personally I don't really see an increase of the security if tokens are hidden and to me it would be more convenient to spare the click as I use 2FA a lot.
Thanks for considering!

**Comment by helloworld1** [comment]
Could be awesome to have also.

**Comment by 6543** [comment]
yes could be an user option - I personally like it hidden by default

**Comment by jacobgkau** [comment]
When the clipboard is enabled, tapping copies to the clipboard; when it's disabled, I can't copy. So I either get my clipboard cluttered up with codes I'm only trying to view (to input on other devices), or I have to go and change the setting every time I want to copy a code (to input on the same device). Allowing showing all tokens would solve this by letting me see the tokens to input on other devices, and tap to copy.
